src/main/java/com/google/api/gax/grpc/ServiceApiSettings.java

```diff
@@ -106,6 +114,16 @@ public ManagedChannel getChannel(Executor executor) throws IOException {
             .build();
         return channel;
       }
+
+      protected String serviceHeader() {
```

Comment:
Using "protected" here doesn't really make sense - the code directly instantiates this class with no provision for a subclass. Use "private".